backends/platform/sdl/psp2/libvita2d/include/bin_packing_2d.h

```diff
@@ -0,0 +1,37 @@
+#ifndef BIN_PACKING_2D_H
```

Comment:
vita2d is a standard library of the vitasdk but it is Frangarcj's "fbo" branch. it can be just downloaded from github and built with "make install" basically. Detailed instructions are on readme-psp2.md. Tldr: this is a "standard" library similar to SDL. No need to include it with scummVM